tsconfig.json

```diff
@@ -20,7 +20,7 @@
     "forceConsistentCasingInFileNames": true,
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "noEmit": false
+    "noEmit": true
```

Comment:
Why noEmit?

Reply:
It's not meant to be commited :P

Reply:
`noEmit` is `false` by default, but if this line is removed, `npm run build` will add a `"noEmit": true` automatically 🤔:

```
The following changes are being made to your tsconfig.json file:
  - compilerOptions.noEmit to be suggested value: true (this can be changed)
```

Reply:
I've never seen that one before. A google search says this is coming from `create-react-app` 🤔 
https://stackoverflow.com/q/53945131/266535

Reply:
I'll just set it to `"noEmit": false`